modules/administration-guide/partials/con_external-database-setup.adoc

```diff
@@ -34,13 +34,14 @@ The recommended configuration of the example database is:
 |Automated backups | enabled (30 days)
 |===
 
-include::partial$proc_configuring-external-PostgreSQL.adoc[leveloffset=+1]
+.Additional resources
```

Comment:
Additional resources should be the last section in a mod docs module. They can't be followed by include statements.

Reply:
@themr0c, :) I don't understand, can you please clarify why additional resources can't be followed by include statements?